build/Common.props

```diff
@@ -13,6 +13,14 @@
     <!--<AnalysisLevel>latest-All</AnalysisLevel>-->
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- https://dotnet.microsoft.com/download/dotnet-core -->
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworksForDocs>net7.0;net6.0</TargetFrameworksForDocs>
+    <!-- https://dotnet.microsoft.com/download/dotnet-framework -->
+    <TargetFrameworksForDocs Condition="$(OS) == 'Windows_NT' And '$(UsingMicrosoftNETSdkWeb)' == 'False'">$(TargetFrameworksForDocs);net462;net47;net471;net472;net48</TargetFrameworksForDocs>
```

Comment:
i know its not introcuded in this PR, but wondering why we need all .NET FW targets, instead of just Net462 alone?

Reply:
This allows the docs examples to build/run under all supported frameworks using the exact runtime version. In other words, these are all TFMs that OpenTelemetry examples support.

Reply:
Should we add `net481`?

Reply:
Yes, let me add that.

Reply:
Added.